### ISSUE TITLE
School_Selection_Bar UX Improvement 

### ISSUE THREAD

**Issue by MANI14011998** [systers/PowerUp#349]
## Description
As a User it's not easy to understand  in what tab I am in  until or unless i will observe the categories so it should show highlighted the button.
So that it will highlight the tab When Ever User Clicks On Selection Bar 
ie-Hair Bar It will Change The Hair Into WhiteBackground Colour
## Mocks
![shop_selection_bar](https://user-images.githubusercontent.com/34857581/75242655-885f4780-57ee-11ea-91c1-3f65bbe22847.png)
some thing like that

### Approach 
- [ ] Approach is to highlight the button 
### Enhancement to Update
- [ ] UI & UX more easy To under stand 

## Definition of Done
- [ ] Approval by 1 mentor.
- [ ] All of the required items are completed.

## Estimation
4 hours

**Comment by MANI14011998** [comment]
@sunjunkie ma'am please review this issue

**Comment by sunjunkie** [comment]
1  - What does "Do this" mean in the description? Please describe what the user wants to do and why. Use the issue template.
2 - The icons you propose, by the way, don't follow the look and feel of the app. A common approach is to highlight the button in some way.

**Comment by MANI14011998** [comment]
Yeah i am not using the icon
It's just an example I will  make the highlighted button
 @sunjunkie  can I work on this issue

**Comment by sunjunkie** [comment]
@MANI14011998 yes.

**Comment by yugantarjain** [comment]
I've encountered this same issue and believe this to be its cause-
The buttons to select the categories(hair/clothes/accessories) are simply placed on top of the shop_category_bar image and are being used as a tap recogniser. The background image contains the text while the buttons are empty, this makes it difficult to highlight the selected category.

By replacing the background image(removing text from it) and displaying that text through the buttons instead, this issue can be easily solved.

**Comment by sunjunkie** [comment]
Yes, we have this issue in a number of places in the app. We do need to separate the buttons from the background entirely. This will also allow flexible placement of the buttons as well as the ability to highlight them. So this issue will require some changes to the existing background image.
